VocableUITests/Screens/SettingsScreen.swift

```diff
@@ -117,8 +118,14 @@ class SettingsScreen: BaseScreen {
         categoriesButton.tap()
     }
     
+    func navigateToTimingAndSensitivityScreen() {
+        _ = mainScreen.settingsButton.waitForExistence(timeout: 2)
+        mainScreen.settingsButton.tap()
+        _ = timingAndSensitivityButton.waitForExistence(timeout: 2)
+        timingAndSensitivityButton.tap()
```

Comment:
```suggestion
        MainScreen.settingsButton.tap(afterWaitingForExistenceWithTimeout: 2)
        timingAndSensitivityButton.tap(afterWaitingForExistenceWithTimeout: 2)
```
I would double check this before committing, but I believe Rudy's latest changes allow for this ^